js/client.js

```diff
@@ -1471,7 +1471,7 @@
 			if (leftWidth < leftMinMain) {
 				leftWidth = leftMinMain;
 			}
-			if (this.curRoom.type === 'battle' && this.sideRoom.type === 'chat') {
+			if (this.curRoom.type === 'battle' && this.sideRoom.type !== 'battle') {
```

Comment:
Oh, at the moment they can't. That may change at a future date but for now this works, I suppose.